free-foil-sprite/src/Language/Sprite/Syntax/Lex.x

```diff
@@ -28,7 +28,7 @@ $u = [. \n]          -- universal: any character
 
 -- Symbols and non-identifier-like reserved words
 
-@rsyms = \( | \) | \= \> | \{ | \} | \/ \* \@ | \: | \* \/ | \= | \; | \+ | \- | \* | \[ | \| | \] | \= \= | \< | \< \= | \,
+@rsyms = \( | \) | \= \> | \{ | \} | \/ \* \@ | \: | \* \/ | \= | \; | \+ | \- | \* | \[ | \| | \] | \= \= | \< | \< \=
```

Comment:
I think these files should be marked as generated in `.gitattributes` file. See [this documentation](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github).